docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -24,11 +26,16 @@ services:
       - MYSQL_DATABASE=opencast
       - MYSQL_USER=opencast
       - MYSQL_PASSWORD=opencast
-    ports:
-      - "3306:3306"
     volumes:
       - ./assets/opencast-ddl.sql:/docker-entrypoint-initdb.d/opencast-ddl.sql:ro
       - db:/var/lib/mysql
+    command: [
+            '--wait_timeout=28800',
+        ]
+    networks:
+      default:
+        aliases:
+          - mariadb
```

Comment:
I don't like this change. This hostname should already be set by default as the service is named `mariadb` (see e.g. <https://docs.docker.com/compose/networking/>). If this is not working for you, this is either a bug (unlikely I have the exact same version running on a Mac) or something is up with your setup.

Reply:
It was problem with the DDL file, rolling back this change